plugins/woocommerce-admin/client/products/sections/product-inventory-section/manual-stock-section.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { RadioControl } from '@wordpress/components';
+import { useFormContext } from '@woocommerce/components';
+import { Product } from '@woocommerce/data';
+
+export const ManualStockSection: React.FC = () => {
+	const { getInputProps, values } = useFormContext< Product >();
```

Comment:
Where else are we using the `values` variable?

Reply:
We're not; probably left over from a previous iteration.  Good catch!